compiler/AST/primitive.cpp

```diff
@@ -1034,6 +1034,10 @@ initPrimitive() {
   // indicate optimization information.
   // That symbol includes OPT_INFO_... flags.
   prim_def(PRIM_OPTIMIZATION_INFO, "optimization info", returnInfoVoid, true, false);
+
+  prim_def(PRIM_POPULATE_TESTS,"populate tests",returnInfoDefaultInt);
```

Comment:
I think I would rename this to "populate tests list" or "gather tests" if the former felt too long.  "Populate tests" could be reasonably interpreted to mean "populate the contents of a test" or something to someone not as familiar with the implementation.

Reply:
Changed the names.